index.js

```diff
@@ -404,15 +404,21 @@ export default function(value, nominatim_object, optional_conf_parm) {
      */
     function getRegexKeyForKeyFromOsmDefaults(key) {
         var regex_key;
+        var exact_match = false;
 
-        for (var osm_key in osm_tag_defaults) {
+        Object.keys(osm_tag_defaults).forEach(function (osm_key) {
+            if (exact_match === true) {
+                return;
+            }
             if (key === osm_key) { // Exact match.
                 regex_key = osm_key;
-                break;
+                // We can't just return here as some old browsers
```

Comment:
Added this comment to avoid to break with some future refactor and until we don't drive browser support/deprecation through a transpiler.